angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/nimble-table-column-anchor-router-link-with-href.directive.ts

```diff
@@ -0,0 +1,51 @@
+import { LocationStrategy } from '@angular/common';
+import { Directive, ElementRef, Inject } from '@angular/core';
+import { ActivatedRoute, Router, RouterLinkWithHref } from '@angular/router';
+import type { TableColumnAnchor } from '@ni/nimble-components/dist/esm/table-column/anchor';
+import type { TableColumnAnchorCellView } from '@ni/nimble-components/dist/esm/table-column/anchor/cell-view';
+import type { DelegatedEventEventDetails } from '../base/nimble-table-column-base.directive';
+
+/**
+ * Directive to handle nimble-table-column-anchor RouterLink support.
+ * Note: Clients need to use [nimbleRouterLink] instead of [routerLink], so that there
+ * won't also be an active RouterLink directive incorrectly handling navigation.
+ */
+@Directive({
+    selector: 'nimble-table-column-anchor[nimbleRouterLink]'
+})
+export class NimbleTableColumnAnchorRouterLinkWithHrefDirective extends RouterLinkWithHref {
+    public constructor(elementRef: ElementRef<TableColumnAnchor>, router: Router, route: ActivatedRoute, @Inject(LocationStrategy) locationStrategy: LocationStrategy) {
+        super(router, route, locationStrategy);
+
+        elementRef.nativeElement.addEventListener('delegated-event', (event: Event) => this.onDelegatedEvent(event));
+    }
```

Comment:
Use a HostListener as discussed in spec

Reply:
Whoops. Had written this already and forgot to update it.